src/navigation/FirstPersonControls.js

```diff
@@ -12,264 +12,252 @@
  *
  *
  */
- 
-Potree.FirstPersonControls = class FirstPersonControls extends THREE.EventDispatcher{
-	
-	constructor(viewer){
+
+Potree.FirstPersonControls = class FirstPersonControls extends THREE.EventDispatcher {
+	constructor (viewer) {
 		super();
-		
+
 		this.viewer = viewer;
 		this.renderer = viewer.renderer;
-		
+
 		this.scene = null;
 		this.sceneControls = new THREE.Scene();
-		
+
 		this.rotationSpeed = 200;
 		this.moveSpeed = 10;
-		
+
 		this.keys = {
-			FORWARD:   ['W'.charCodeAt(0), 38],
-			BACKWARD:  ['S'.charCodeAt(0), 40],
-			LEFT:      ['A'.charCodeAt(0), 37],
-			RIGHT:     ['D'.charCodeAt(0), 39],
-			UP:        ['R'.charCodeAt(0), 33],
-			DOWN:      ['F'.charCodeAt(0), 34]
+			FORWARD: ['W'.charCodeAt(0), 38],
```

Comment:
same here, would be nice to be able to preserve the alignment of key / values in form of a table,